src/pages/AnimeOnsen/style.less

```diff
@@ -32,6 +32,10 @@ span.malp-group-label,
   border-color: currentcolor;
 }
 
+main div.container {
+  grid-template-rows: max-content repeat(2, auto) 1fr;
```

Comment:
What is the idea behind this styling?

Reply:
> What is the idea behind this styling?

The page styling gets broken if the `malp` group label thingy gets inserted into the page. Because the details page has a specific `grid-template-rows` sizing, if a new element gets inserted in the grid, it will have malformed styling (in this case, `malp` will get `1fr` grid row size instead of `auto` and as such the elements of the description page (such as episode list) has no row size and is defaulted to minimum row size.